.rubocop_todo.yml

```diff
@@ -339,6 +339,7 @@ Performance/StringReplacement:
 # Cop supports --auto-correct.
 Security/YAMLLoad:
   Exclude:
+    - 'lib/cocoapods/sources_manager.rb'
```

Comment:
Suggestion: exclude the specific line and not the entire file.

Reply:
Fixed f4430258c09af203cd3ea565ed4a60f9ce89a2dc